libraries/classes/Version.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpMyAdmin;
+
+use function defined;
+
+/**
+ * Class to handle the phpMyAdmin version
+ */
+final class Version
+{
+    /*
+     * Packaging people can add a version suffix using the VERSION_SUFFIX constant at vendor_config.php
+     */
+
+    public const VERSION_MAJOR = 5;
+    public const VERSION_MINOR = 1;
+    public const VERSION_PATCH = 0;
+
+    // The version will be {major}.{minor}.{patch}-dev
+    public const IS_DEV = true;
+
+    // The version will be {major}.{minor}.{patch}-{PRE_RELEASE_NAME}
+    public const IS_PRE_RELEASE = false;
+
+    public const PRE_RELEASE_NAME = '';
```

Comment:
```suggestion
    public const PRE_RELEASE_NAME = 'rc1';
```

https://github.com/phpmyadmin/phpmyadmin/blob/ffbebaa8a27e646e90d658515e703da9fbb2596b/libraries/classes/Config.php#L139

Reply:
This will have to be filled in when releasing, for now it should be empty